### DISCUSSION TITLE
惩治倒卖本项目源代码！！！

### DISCUSSION THREAD

**Discussion by ProgramCX** [win12-online/win12#508] [General]
【闲鱼】https://m.tb.cn/h.TWUbIbh?tk=ThNb3Fq0nvk CZ3458 「快来捡漏【Windows 12 网页版源码，在线体验：https://】」
点击链接直接打开
![Screenshot_2024-12-03-21-39-45-23_0dff84d2da4d0ad](https://github.com/user-attachments/assets/5e52f28d-2574-43b1-bab0-b8ffaabd2d8d)

暑假的时候举报了，但是失败了。😡😡😡

**Comment by ProgramCX** [comment]
跟他没完。开源项目在国内保护就这么差吗😭

**Reply by lingbopro** [reply]
貌似是的...

**Reply by lingbopro** [reply]
![EPL rules](https://github.com/user-attachments/assets/b0fecbfa-da55-4d14-9762-cc962ce7f44c)

他显然没有遵守EPL

**Reply by ProgramCX** [reply]
是的。但是没办法惩治他😭起诉？

**Reply by Mbilse** [reply]
> ![EPL rules](https://github.com/user-attachments/assets/b0fecbfa-da55-4d14-9762-cc962ce7f44c)
> 
> 他显然没有遵守EPL

换一个？

**Comment by kjmjh** [comment]
还卖两块八😡😡😡😡😡😡😡😡😡😡😡😡😡😡😡

**Reply by lingbopro** [reply]
两块八确实有点多了（）

**Reply by kjmjh** [reply]
学牲没有那么多钱😡😡😡😡😡😡😡😡😡😡😡😡，况且这开源项目😡😡😡😡😡😡😡😡😡😡😡😡举报去😡😡😡😡😡😡😡😡😡😡

**Reply by Mbilse** [reply]
> 学牲没有那么多钱😡😡😡😡😡😡😡😡😡😡😡😡，况且这开源项目😡😡😡😡😡😡😡😡😡😡😡😡举报去😡😡😡😡😡😡😡😡😡😡

成功了吗？

**Reply by kjmjh** [reply]
找不到在哪举报。。

**Reply by Mbilse** [reply]
> 找不到在哪举报。。

消费者投诉
试一下？

**Reply by kjmjh** [reply]
初中生？

**Reply by lingbopro** [reply]
> 初中生？

？

**Reply by kjmjh** [reply]
？

**Reply by Mbilse** [reply]
> 初中生？

我才五年)qaq

**Reply by kjmjh** [reply]
awa

**Comment by kj23qaq** [comment]
呕吼，又来一个[某宝](https://item.taobao.com/item.htm?spm=a21n57.sem.item.57.5bf617c5FDLsMv&priceTId=214783ff17408151494085067eba76&utparam=%7B%22aplus_abtest%22%3A%22a6be2a5d92895644688ae27d419a7c49%22%7D&id=837461093630&ns=1&abbucket=17&xxc=taobaoSearch&pisk=gnQjxKY8bGaoaX2GqtVyNFDIdaYsX7zUGfOOt13q6ELxW56NUN-9QoX6fOWkXZ8VgT9OLT9x7lvq5F11IOW1sSA9BOB6Qirz8sfDSFeFCyzFiKKCUwWXDCH86COtB7-xYMBClFeULPirwn22WtPKerh-2BvJMCLtW7GJiCL96ILTNLd6tcHABOC8NCdMMcKxDLCJiB8tBK3vw0dw6j3v6KC8NCvJBFLOBUdKS8ASldfbMldn36g736vSWVQWwJtPGl0MwKRfSL59HV3TsItXpspuK9GHLH6vfNVIPTsAnZTyAfGWWiCCPM1jM455MZXf4k3edfASfcvxWQ9UN7iijlkSbKXfg8hpDQAlJ7NSvdxvZQ9UN7iijndkwuV7NDpG.)

**Reply by kj23qaq** [reply]
几个更nb的:https://www.goofish.com/item?spm=a21ybx.search.searchFeedList.4.72257e1aVMh8b3&id=834863281954&categoryId=201452811

https://www.goofish.com/item?spm=a21ybx.search.searchFeedList.3.72257e1aVMh8b3&id=834247582150&categoryId=201453616

**Reply by ProgramCX** [reply]
![image](https://github.com/user-attachments/assets/29d2910a-02f4-40cf-857f-197bcf84a043)
“很多没用的功能“？？？？？？？？？？？？？？？？？？？？？？？？？？？有点想干他了

**Reply by kjmjh** [reply]
《PE》《加载引导》《win12web》《虚拟机》

**Reply by Mbilse** [reply]
要登录,我没帐号！
![Screenshot_2025-03-01-18-50-17-60.jpg](https://github.com/user-attachments/assets/fc3a86d5-4e83-44d7-a427-c06879128923)

！！！

**Comment by tangyuan0821** [comment]
## 若平台需要投诉者描述事件过程或投诉理由。可参考以下示例
XX 人在贵平台售卖的 XX 商品中，包含 Windows 12 网页版（网址：https://github.com/tjy-gitnub/win12）的计算机程序源代码。

我方认为，相关商品可能构成对 Eclipse Public License 2.0 许可证的滥用，包括：

- 其未充分尊重 Windows 12 网页版项目作者及相关贡献者的署名权，故意隐瞒了原项目中的署名信息（谭景元，tjy-gitnub）和项目链接（https://github.com/tjy-gitnub/win12）；

- 其提供的计算机程序源代码中可能不包含 Windows 12 网页版的许可证及版权声明，违反了许可证中关于不得移除、隐藏开源声明的要求。

因此，我方作为该项目的相关权利人，认为该商品侵害了我方合法权益，现要求贵平台下架相关商品。

**我方在此承诺：**

我方确认相关商品内容或链接侵犯我方的民事权利，投诉行为符合中华人民共和国（不含香港特别行政区、澳门特别行政区、台湾地区）法律规定及社会公序良俗，投诉通知中所有内容均真实、完整、合法、有效。

如本投诉通知存在任何不完整、不实、不合法或不符合社会公序良俗之处，我方将承担由此产生的一切法律责任。

Windows 12 网页版项目相关权利人

XXXX 年 XX 月 XX 日

**Reply by tangyuan0821** [reply]
这个示例适用于大部分倒卖项目源代码的商品。只需要把文中的"XX"分别替换一下就可以了。

**Comment by lingbopro** [comment]
但举报是一回事，平台管不管就是另一回事了…
